sdk-base/src/test/java/com/mapbox/maps/util/MathUtilsTest.kt

```diff
@@ -1,42 +1,60 @@
 package com.mapbox.maps.util
 
+import com.mapbox.maps.util.MathUtils.prepareOptimalBearingPath
+import com.mapbox.maps.util.MathUtils.shortestRotation
 import org.junit.Assert.assertArrayEquals
 import org.junit.Assert.assertEquals
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
 
-internal class MathUtilsTest {
+private const val EPS = 0.0001
```

Comment:
put in companion object block?

Reply:
no need I guess. I used this constant in two different classes inside one file